parachain/pallets/ethereum2-light-client/src/weights.rs

```diff
@@ -0,0 +1,59 @@
+//! Autogenerated weights for ethereum_light_client
```

Comment:
Benchmarks won't be important for quite a while. We can add them much later. So I would advise just removing this file and any other benchmarking stuff,